### PR TITLE
fix some minor suggestions and warnings and deprecations

### PR DESCRIPTION
**Pull request by Yolgie**

### CODE REVIEW

File: eventcollectors/build.gradle.kts [hunk 1]
```diff
@@ -15,6 +15,7 @@ dependencies {
         exclude("org.codehaus.groovy", "groovy-dateutil")
     }
     implementation("com.rometools:rome:2.1.0")
+    implementation("org.apache.httpcomponents:httpclient:4.5.13")
```
Comment (kadhonn) [inline eventcollectors/build.gradle.kts:18]: why did you add this dependency explicitly?

File: eventcollectors/src/main/kotlin/events/boudicca/eventcollector/collectors/TechnologiePlauscherlCollector.kt [hunk 2]
```diff
@@ -49,4 +51,14 @@ class TechnologiePlauscherlCollector : EventCollector {
 
         return events
     }
+
+    private fun prepareSyndFeed(url: String): SyndFeed {
+        HttpClients.createMinimal().use { client ->
```
Comment (kadhonn) [inline eventcollectors/src/main/kotlin/events/boudicca/eventcollector/collectors/TechnologiePlauscherlCollector.kt:56]: please use our Fetcher class for all http relevant stuff, as it implements a delay handling, sets the user-agent to boudicca and records http calls automatically in our collectors.boudicca.event overview

**Review by kadhonn — CHANGES_REQUESTED** [review]

File: eventcollectors/build.gradle.kts (hunk 1 above)
Comment (kadhonn) [reply eventcollectors/build.gradle.kts:18]: because you used it below...

File: eventcollectors/src/main/kotlin/events/boudicca/eventcollector/collectors/TechnologiePlauscherlCollector.kt (hunk 2 above)
Comment (Yolgie) [reply eventcollectors/src/main/kotlin/events/boudicca/eventcollector/collectors/TechnologiePlauscherlCollector.kt:56]: the XmlReader with an url as param has been deprecated for quite some time, if you can create a synd feed with the Fetcher, pls tell me how to do it, otherwise this would require major refactoring of either the Fetcher or the Techplauscherl Collector

Comment (Yolgie) [reply eventcollectors/src/main/kotlin/events/boudicca/eventcollector/collectors/TechnologiePlauscherlCollector.kt]: i just converted the fetched content to an InputStream, not the most elegant solution, but it works...

**Review by kadhonn — APPROVED** [review]
looks good, thanks for the cleanup!
